### PR TITLE
Remove architecture suffix from nvdaControllerClient.dll

### LINKED ISSUE #15717: Controller client library for ARM64 is called nvdaControllerClient32.dll
### Steps to reproduce:
1. run `scons client`
2. Go to `extras\controllerClient\arm64`

### Actual behavior:
Resulting ddl is called `nvdaControllerClient32.dll`

### Expected behavior:
Something diffferent

### Additional context
The library is called `nvdaControllerClient64.dll` for x86_64, otherwise it is called `nvdaControllerClient32.dll`.

### Proposed fix
I propose removing the architecture suffix from the dll name. The architecture is clear from the folder the dll is bundled in. Furthermore, X64 examples like the C# and Python ones don't run on X64 without changes.

### PR DESCRIPTION
**Pull request by LeonarddeR**
### Link to issue number:
Fixes #15717 

### Summary of the issue:
The controller client library for ARM64 is called nvdaControllerClient32.dll

### Description of user facing changes
None, devs only change

### Description of development approach
Remove the 32/64 suffix from nvdaControllerClient32/64.dll. This change also has the benefit that python/c# samples run without change for X64 versions of the dll.

### Testing strategy:
Only file name changes.

### Known issues with pull request:
None known
### Code Review Checklist:

- [x] Documentation:
  - Change log entry
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] API is compatible with existing add-ons.
- [x] Security precautions taken.

### CODE REVIEW

File: extras/controllerClient/x86/example_python.py [hunk 1]
```diff
@@ -1,20 +1,20 @@
-import time
 import ctypes
+import time
```
Comment (seanbudd) [inline extras/controllerClient/x86/example_python.py:2]: can this get copyright headers added?

File: user_docs/en/changes.t2t [hunk 2]
```diff
@@ -56,6 +56,7 @@ Add-ons will need to be re-tested and have their manifest updated.
 - The ``BrailleDisplayDriver`` base class now has ``numRows`` and ``numCols`` properties to provide information about multi line braille displays.
 Setting ``numCells`` is still supported for single line braille displays and ``numCells`` will return the total number of cells for multi line braille displays. (#15386)
 - Updated BrlAPI for BRLTTY to version 0.8.5, and its corresponding python module to a Python 3.11 compatible build. (#15652, @LeonarddeR)
+- The file names of the NVDA Controller Library no longer contain a suffix denoting the architecture, i.e. ``nvdaControllerClient32/64.dll`` are now called ``nvdaControllerClient.dll``. (#15718, #15717, @LeonarddeR)
```
Comment (seanbudd) [inline user_docs/en/changes.t2t:59]: should this be under API breaking changes?

Comment (leonardder) [reply user_docs/en/changes.t2t:59]: I don't think so. This doesn't break a particular API, i.e. the old controller client can still be used and nothing is changed about the internals of the controller, so people don't need updating. The only thing that changes is the distribution.

Comment (leonardder) [reply user_docs/en/changes.t2t:61]: Actually I'm planning to provide a follow up that will be API breaking anyway, so we can revisit the changelog as part of that pr. Is that ok with you? Code is mostly written but blocked by #15699

**Review by seanbudd — APPROVED** [review]
Thanks @leonardder

**Pull request merged**